src/style.css

```diff
@@ -2,0 +1,60 @@
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-bar__form {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.search-bar__item::placeholder {
+  color: rgba(61, 78, 97, 1);
+}
+
+.search-bar__item {
+  display: block;
+
+  margin-top: 20px;
+  width: 100%;
+  height: 42px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+
+  background-image: url(images/Search.svg);
```

Comment:
Ensure that the `src/images` directory is correctly utilized as per task requirements. Verify the presence of the `images/Search.svg` file in the specified directory.